src/localize.cpp

```diff
@@ -319,76 +300,80 @@ void localize_wrk(const BasisSet & basis, arma::subview<double> & C, arma::subvi
       // and energies
       arma::vec Eloc(Cloc.n_cols);
       if(H.n_rows == Cwrk.n_rows) {
-	// We have Fock operator; use it to calculate energies (in case
-	// we don't have canonical orbitals here)
-	for(size_t io=0;io<Cloc.n_cols;io++)
-	  Eloc(io)=arma::as_scalar(arma::trans(Cloc.col(io))*H*Cloc.col(io));
+	  // We have Fock operator; use it to calculate energies (in case
+	  // we don't have canonical orbitals here)
+	  for(size_t io=0;io<Cloc.n_cols;io++)
+	    Eloc(io)=arma::as_scalar(arma::trans(Cloc.col(io))*H*Cloc.col(io));
```

Comment:
This for loop could also be written as `Eloc = arma::diagvec(arma::trans(Cloc)*H*Cloc);`

Reply:
Yes, I have implemented that change